packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -285,14 +377,18 @@ export default declare((api, options) => {
             } = path.node;
 
             if (isStatic) {
-              throw path.buildCodeFrameError(
-                "Static class fields are not spec'ed yet.",
-              );
+              if (privateStaticNames.has(name)) {
```

Comment:
Private names are shared between instance and static, ie, the following is an error:

```js
class Example {
  static #x;
  #x;
}
```

Reply:
Wasn't sure about this one, I'll make the change.